### PR TITLE
Update adviser promo in train to teach section

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/3Bweg0NG/7061

### CODE REVIEW

File: app/views/content/train-to-be-a-teacher/school-placements.md [hunk 1]
```diff
@@ -20,8 +20,9 @@ quote:
     text: "My mentor was the special educational needs co-ordinator. This meant I was able to learn more about this role and improve my understanding of the strategies and support available for children with special educational needs and disabilities (SEND)."
     name: "Abbe, trainee teacher"
     classes: quote--indent
-promo_content:
-    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+cta_adviser:
+  adviser:
+    text: An adviser with years of teaching experience can tell you more about school placements and how to submit a successful application. Chat by phone, text or email, as little or as often as you need.
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/school-placements.md:25]: I don't think we can say `how to submit a successful application` as that depends on the candidate's eligibility etc. etc. (which is why we talk about `Maximise your chances of success.....` in other promos

File: lib/tasks/support/teaching-internship-providers.md.erb [hunk 2]
```diff
@@ -5,8 +5,6 @@ subcategory: Postgraduate teacher training
 description: |-
   Find paid teaching internships to gain new skills and see what classroom life is like. Explore chemistry, computing, languages, maths and physics internships.
 image: false
-promo_content:
```
Comment (gemmadallmandfe) [inline lib/tasks/support/teaching-internship-providers.md.erb:8]: Does the cta_adviser need adding here too (you've added it to the AO page `lib/task`)?

One for @martyn-w to advise on!

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md [hunk 3]
```diff
@@ -6,8 +6,9 @@ description: |-
   Find out about the assessment only route to qualified teacher status (QTS) for unqualified teachers who have worked in a classroom.
 image: false
 backlink: /
-promo_content:
-    - content/train-to-be-a-teacher/promos/adviser-promo-assessment-only
+cta_adviser:
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:9]: I can't see this on the front end? Check with Martyn as to how this needs to work!

File: app/views/content/train-to-be-a-teacher/school-placements.md (hunk 1 above)
Comment (emmaachesongray) [inline app/views/content/train-to-be-a-teacher/school-placements.md:25]:
```suggestion
    text: An adviser with years of teaching experience can tell you more about school placements and how to maximise your chances of submitting a successful application. Chat by phone, text or email, as little or as often as you need.
```

File: app/views/content/train-to-be-a-teacher/teaching-internship-providers/_listing.html.erb [hunk 4]
```diff
@@ -70,3 +70,8 @@
 <section class="providers-list">
   <%= render GroupedCards::ListingComponent.new @front_matter["provider_groups"] %>
 </section>
+<section>
+  <%= render CallsToAction::AdviserComponent.new(
+    text: "An explore teaching adviser can help you find school experience, learn what it's like to be in the classroom and answer all your questions about getting into teaching.",
```
Comment (emmaachesongray) [inline app/views/content/train-to-be-a-teacher/teaching-internship-providers/_listing.html.erb:75]:
```suggestion
    text: "An explore teaching adviser can help you find school experience, learn what it's like to be in the classroom and answer all your questions about getting into teaching.",
    link_target: "/explore-teaching-advisers",
```

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md (hunk 3 above)
Comment (martyn-w) [reply app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:9]: @emmaachesongray in addition to defining the adviser promo here in the frontmatter, you also need to call the adviser promo in the markdown (at the bottom of this file) with `$adviser$`

Comment (martyn-w) [reply app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:9]: Furthermore, this file is generated from a template, and the template will need to be updated too: `lib/tasks/support/assessment-only-route-to-qts.md.erb`

File: lib/tasks/support/assessment-only-route-to-qts.md.erb [hunk 5]
```diff
@@ -6,8 +6,9 @@ description: |-
   Find out about the assessment only route to qualified teacher status (QTS) for unqualified teachers who have worked in a classroom.
 image: false
 backlink: /
-promo_content:
-    - content/train-to-be-a-teacher/promos/adviser-promo-assessment-only
+cta_adviser:
+  adviser:
+    text: An adviser with years of teaching experience can talk you through the assessment only programme. Chat through phone, text or email, as little or as often as you need.
```
Comment (martyn-w) [inline lib/tasks/support/assessment-only-route-to-qts.md.erb:11]: Don't forget to add a `$adviser$` to the markdown section of this file

File: lib/tasks/support/teaching-internship-providers.md.erb (hunk 2 above)
Comment (martyn-w) [reply lib/tasks/support/teaching-internship-providers.md.erb:8]: @emmaachesongray @gemmadallmandfe my reading of this PR is the new adviser promo _will_ be displayed as it has been added to `app/views/content/train-to-be-a-teacher/teaching-internship-providers/_listing.html.erb` - and so does not need to be added to this file.

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts/_update-details.erb [hunk 6]
```diff
@@ -3,3 +3,6 @@
     If you're a provider and need to update the details on your record, please contact us at: <%= mail_to("getintoteaching.helpdesk@education.gov.uk") %>.
   </p>
 </section>
+  <%= render CallsToAction::AdviserComponent.new(
+    text: "An adviser with years of teaching experience can talk you through the assessment only programme. Chat by phone, text or email, as little or as often as you need.",
+) %>
```
Comment (martyn-w) [inline app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts/_update-details.erb:8]: maybe consider a `border: false` parameter too?

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

Does this need a tech review as well though?

**Review by martyn-w — APPROVED** [review]
👍  LGTM

**Pull request merged**